runtime/lua/vim/lsp/util.lua

```diff
@@ -121,50 +121,61 @@ local function get_line_byte_from_position(bufnr, position)
 end
 
 --- Process and return progress reports from lsp server
-function M.get_progress_messages()
+function M.get_progress_messages(bufnr)
```

Comment:
I would check if bufnr is nil, and if so use the currently active buffer.

Reply:
Yeah, I can add `bufnr = bufnr or 0`